python/ray/train/torch/torch_checkpoint.py

```diff
@@ -74,6 +75,7 @@ def from_state_dict(
         state_dict: Dict[str, Any],
         *,
         preprocessor: Optional["Preprocessor"] = None,
+        training_state: Optional[Dict[str, Any]] = None,
     ) -> "TorchCheckpoint":
```

Comment:
Should this be added to other `Checkpoint` types as well?

Reply:
I don't think so. AFAIK this functionality isn't necessary for other frameworks.

Reply:
Why is this not needed for other frameworks?